kotlin-jvm/settings.gradle.kts

```diff
@@ -1,5 +1,15 @@
 import de.fayard.refreshVersions.bootstrapRefreshVersions
 
+pluginManagement {
+    repositories {
+        mavenLocal()
+        mavenCentral()
+        jcenter()
+        google()
+        gradlePluginPortal()
+    }
+}
```

Comment:
@jmfayard Can you take care of removing these? They are not needed.

Reply:
I will!

Reply:
I have

Reply:
Great!